metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -736,6 +748,37 @@ def standardize_schema_table_names(
             raise ValueError(f"schema {schema} does not match table {entity}")
         return segments[0], segments[1]
 
+    def gen_schema_key(self, db_name: str, schema: str) -> PlatformKey:
+        return BigQuerySchemaKey(
+            project_id=db_name,
+            schema=schema,
+            platform=self.platform,
+            instance=self.config.env,
+        )
+
+    def gen_database_key(self, database: str) -> PlatformKey:
+        return ProjectIdKey(
+            project_id=database,
+            platform=self.platform,
+            instance=self.config.env,
+        )
+
+    def gen_database_containers(self, database: str) -> Iterable[MetadataWorkUnit]:
```

Comment:
Should we call this "gen_project_containers"?

Reply:
In the bigquery world there is no such thing like database:
Database == Project
Schema == Schema (or dataset)

Reply:
Here this method overrides the one in sql_common -> In python there is no way to mark @override -> maybe a comment would make sense here?